app/assets/stylesheets/generals.scss

```diff
@@ -0,0 +1,305 @@
+$vivid-blue: #93c5ea;
+$full-white: #ffffff;
+$material-red: #ef9a9a;
+$grayish-cyan: #aebcb9;
+$default-blue: #0b2538;
+
+.default-color {
+  background-color: $default-blue !important;
+}
+
+.default-color-text {
+  color: $full-white;
+}
+
+.icon-block {
+  padding: 0 15px;
+}
+
+#intro,
+#work,
+#team {
+  padding-top: 4rem;
+}
+
+
+#index-banner {
+  background-color: $vivid-blue;
+  background-image: image-url("trim_image");
+  background-position: center;
+  background-repeat: no-repeat;
+  max-height: 864px;
+  min-height: 632px;
+  position: relative;
+}
+
+#nav-f {
+  -webkit-box-shadow: none !important;
+  box-shadow: none !important;
+}
+
+.text-h {
+  color: $full-white;
+  font-size: 6em;
+  font-weight: 100;
+  padding: 15% 0;
+}
+
+.brand-logo {
+  color: $full-white;
+  display: inline-block;
+  font-size: 2.1rem;
+  font-style: normal;
+  font-weight: 100;
+  letter-spacing: 7px;
+  padding: 0;
+  position: absolute;
+}
+
+.text-h2 {
+  font-weight: 100;
+  line-height: 4.5rem;
+  margin-bottom: 4%;
+}
+
+.span-h2 {
+  color: $vivid-blue;
+  font-weight: 300;
+}
+
+.text-b {
+  color: $vivid-blue;
+}
+
+.in {
+  font-style: normal !important;
+  font-weight: 400 !important;
+}
+
+.promo i {
+  color: $vivid-blue;
+  display: block;
+  font-size: 7rem;
+}
+
+.card-content a {
+  color: $vivid-blue;
+}
+
+.card-content a:hover {
+  color: $vivid-blue;
+}
+
+#work,
+#team {
+  background: rgb(247, 247, 247);
```

Comment:
Color literals like `rgb(247, 247, 247)` should only be used in variable declarations; they should be referred to via variable everywhere else.